source/optional.d.ts

```diff
@@ -0,0 +1,14 @@
+/**
+Makes a type optional. Allows a type to be `undefined`, but not `null`.
+
+@example
+```
+import {Optional} from 'type-fest';
+
+type Value = number;
+type OptionalValue = Optional<Value>;
+
+const values: [Value, OptionalValue] = [2, undefined];
```

Comment:
> Example code block with a realistic example. - https://github.com/sindresorhus/type-fest/blob/master/.github/contributing.md

This example is not really realistic. A more realistic example would be to use `OptionalValue` as a return value or parameter for a function.